experanto/data.py

```diff
@@ -3,20 +3,36 @@
 from collections import namedtuple
 from collections.abc import Iterable
 from pathlib import Path
+
 import numpy as np
 import torch
 from torch.utils.data import Dataset
 
 from .experiment import Experiment
-from .interpolators import ImageTrial
+from .interpolators import ImageTrial, VideoTrial
 
 
 class Mouse2pChunkedDataset(Dataset):
-    def __init__(self, root_folder: str, sampling_rate: float, chunk_size: int) -> None:
+    def __init__(
+        self,
+        root_folder: str,
+        sampling_rate: float,
+        chunk_size: int,
+        rescale: bool = False,
+        keep_nans: bool = False,
+        interpolation_mode: str = "linear",
+        interp_window: int = 5,
+    ) -> None:
         self.root_folder = Path(root_folder)
         self.sampling_rate = sampling_rate
         self.chunk_size = chunk_size
-        self._experiment = Experiment(root_folder)
```

Comment:
I think here the cleanest way is to have a config dict for all Interpolators.
The config dict would be an input kwarg to the Dataset, and then passed to Experiment and down to the interpolators.

e.g.:
`interp_config = {"screen": {...}, {"eye_tracker": {...}, }`

and then it'd look like this: 
`self._experiment = Experiment(root_folder, interp_config)`

Otherwise the args/kwargs for Experiment() are out of control when more devices and functionality are added